### DISCUSSION TITLE
Asynchronous request

### DISCUSSION THREAD

**Discussion by LinzeSu** [drakkan/sftpgo#1148] [Q&A] [answered]
Hi,
I'd like to know if it's ok to use asynchronous request to the sftpgo server. If not, what is the possible phenomenon that's expected to be seen (Like, all request except the first one fail, etc.) . 
Thanks in advance!

**Comment by drakkan** [comment, marked as answer]
Hello, 

SFTPGo itself has no limitations, but if you upload to the same file concurrently the behaviour depends on the storage backend
